website/.changeset/config.js

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
This file and `website/.changeset/README.md` shouldn't exist

Reply:
Ahh I must have run change set in the website dir :(